index.html

```diff
@@ -9,13 +9,21 @@
 </head>
 
 
-<body>   
-
+<body>
+    <div class = "Container">  
+        <h1>My To Do App</h1>
+        <div class = "inputs"> 
+            <input type = "text" placeholder="Enter Your Task Here" id = "inp">
+            <button onclick = "Add()"> Add </button>
```

Comment:
Functions names should be meaningful as to what it does.

```suggestion
            <button onclick="addTodo"> Add </button>
```